<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcList</class>
 <widget class="QWidget" name="ProcList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bKill">
       <property name="text">
        <string>Kill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bStrace">
       <property name="text">
        <string>strace</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
